src/brs/util/Convert.java

```diff
@@ -49,38 +40,23 @@ public static long parseUnsignedLong(String number) {
   }
 
   public static long parseAccountId(String account) {
-    if (account == null) {
-      return 0;
-    }
-    account = account.toUpperCase();
-    if (account.startsWith("BURST-")) {
-      return Crypto.rsDecode(account.substring(6));
-    } else {
-      return parseUnsignedLong(account);
-    }
+    BurstAddress address = BurstAddress.fromEither(account);
```

Comment:
`fromEither`, you thought of everything

Reply:
yep :) parses as a number or an RS depending on input